File: C#/BluffinMuffin.Protocol/Game/DiscardRoundEndedCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using BluffinMuffin.Protocol.DataTypes;
using BluffinMuffin.Protocol.DataTypes.Attributes;

namespace BluffinMuffin.Protocol.Game
{
    /// <summary>
    /// This command is issued by the server when a discard round is ending, after all players have sent a PlayerDiscardActionCommand that fits the discard round limitations.
    /// </summary>
    public class DiscardRoundEndedCommand : AbstractGameCommand
    {
        /// <summary>
        /// The minimum number of cards that a player must discard
        /// </summary>
        [ExampleValues(1)]
        public DiscardInfo[] CardsDiscarded { get; set; }
    }
}

File: Python/BluffinMuffin.Protocol/game/discard_round_ended_command.py
from abstract_game_command import AbstractGameCommand
from data_types.discard_info import DiscardInfo

__author__ = 'ericmas001@gmail.com'



class DiscardRoundEndedCommand(AbstractGameCommand):
    def __init__(self, obj):
        super().__init__(obj)
        self.cards_discarded = [DiscardInfo(x) for x in obj['CardsDiscarded']]

    def __str__( self ):
        return '{0} [{1}]'.format(super().__str__(), ', '.join([x.__str__() for x in self.cards_discarded]))
